oak_functions/examples/weather_lookup/README.md

```diff
@@ -57,6 +57,42 @@ radius of 40km. See https://s2geometry.io/devguide/examples/coverings for more
 information on coverings. Cell coverings for areas can be visualised using
 https://s2.sidewalklabs.com/regioncoverer/.
 
+### File format
+
+The weather file format looks as follows:
+
+Weather file is a Protobuf compiled file that consists of a list of the
+following entries:
+
+```protobuf
```

Comment:
perhaps add a link to proto file where this is actually defined in the code base?

Reply:
Done